### PR TITLE
DOCS-987: Fixing prometheus metrics to pull from Minio Server docs list

### LINKED ISSUE #987: [RELEASE] minio RELEASE.2023-08-31T15-31-16Z
MinIO Server [RELEASE.2023-08-31T15-31-16Z](https://github.com/minio/minio/releases/tag/RELEASE.2023-08-31T15-31-16Z) has changes that impact the docs:

- [ ] Refactor of replication metrics and moved from bucket to cluster level ([PR #17293](https://github.com/minio/minio/pull/17293))
- [x] Adds replication metrics Grafana dashboard ([PR #17951](https://github.com/minio/minio/pull/17951))

### PR DESCRIPTION
**Pull request by ravindk89**
Staged: http://192.241.195.202:9000/staging/DOCS-987/linux/operations/monitoring/metrics-and-alerts.html

Closes #987 

Two major goals here

1. POC for pulling in markdown files from github.com/minio/minio
2. experiment with `csplit` for ripping portions of those markdown files apart for ease-of-ingest

A few notes:

- Table formatting is not great, but I cannot see a way around this for now
- We are no longer linking directly to metrics, but the groupings should still help
- @rushenn if you have ideas for forcing markdown tables to fixed with based on CSS rules or some other clever way to deal with long monospaced text I'd love to hear it

Thanks all

### CODE REVIEW

**Comment by feorlen** [comment]
Is this  warning expected from a local build?
```
operations/monitoring/metrics-and-alerts.rst:39: WARNING: undefined label: 'minio_available-bucket-metrics'
```

File: source/operations/monitoring/metrics-and-alerts.rst [hunk 1]
```diff
@@ -15,15 +15,37 @@ Metrics and Alerts
...
+
+      Replace the ``HOSTNAME:PORT`` with the :abbr:`FQDN (Fully Qualified Domain Name)` and port of the MinIO deployment.
+      For deployments with a load balancer managing connections between MinIO nodes, specify the address of the load balancer.
+
+
+   .. tab-item:: Bucket Metrics
+
+      .. versionchanged:: RELEASE.2023-08-31T15-31-16Z
+
+      You can scrape :ref:`bucket-level metrics <minio_available-bucket-metrics>` using the following URL endpoint:
```
Comment (feorlen) [inline source/operations/monitoring/metrics-and-alerts.rst:39]: This is what caused that build warning
```suggestion
      You can scrape :ref:`bucket-level metrics <minio-available-bucket-metrics>` using the following URL endpoint:
```

File: source/operations/monitoring/metrics-and-alerts.rst [hunk 2]
```diff
@@ -15,15 +15,37 @@ Metrics and Alerts
...
+   .. tab-item:: Cluster Metrics
+
+      You can scrape :ref:`cluster-level metrics <minio-available-cluster-metrics>` using the following URL endpoint:
+
+      .. code-block:: shell
+         :class: copyable
+
+         http://HOSTNAME:PORT/minio/v2/metrics/cluster
+
+      Replace the ``HOSTNAME:PORT`` with the :abbr:`FQDN (Fully Qualified Domain Name)` and port of the MinIO deployment.
```
Comment (feorlen) [inline source/operations/monitoring/metrics-and-alerts.rst:31]: maybe drop the article
```suggestion
      Replace ``HOSTNAME:PORT`` with the :abbr:`FQDN (Fully Qualified Domain Name)` and port of the MinIO deployment.
```

File: source/operations/monitoring/metrics-and-alerts.rst [hunk 3]
```diff
@@ -15,15 +15,37 @@ Metrics and Alerts
...
+      .. versionchanged:: RELEASE.2023-08-31T15-31-16Z
+
+      You can scrape :ref:`bucket-level metrics <minio_available-bucket-metrics>` using the following URL endpoint:
+
+      .. code-block:: shell
+         :class: copyable
+
+         http://HOSTNAME:PORT/minio/v2/metrics/bucket
+
+      Replace the ``HOSTNAME:PORT`` with the :abbr:`FQDN (Fully Qualified Domain Name)` and port of the MinIO deployment.
```
Comment (feorlen) [inline source/operations/monitoring/metrics-and-alerts.rst:46]:
```suggestion
      Replace ``HOSTNAME:PORT`` with the :abbr:`FQDN (Fully Qualified Domain Name)` and port of the MinIO deployment.
```

**Comment by feorlen** [comment]
Still meh for small windows/screens, but we had  that conversation already about tables. Looks as expected otherwise.

**Review by feorlen — APPROVED** [review]
Aspirational approve, pending the typo'd ref link. One nit for your consideration.

File: source/operations/monitoring/metrics-and-alerts.rst (hunk 1 above)
Comment (ravindk89) [reply source/operations/monitoring/metrics-and-alerts.rst:39]: Aha. Underscores.

**Pull request merged**